app/views/projects/new.html.slim

```diff
@@ -6,15 +6,15 @@ nav.page
 
 p.well Please make sure you have read our #{link_to "mentor guide", 'http://railsgirlssummerofcode.org/guide/projects/'} before you submit your Open Source project for the upcoming Rails Girls Summer of Code.
 
-= simple_nested_form_for @project, data: { js: "project-form" } do |f|
+= simple_nested_form_for @project, data: { js: "form" } do |f|
```

Comment:
Is it better to name it `"form-with-preview"` instead of just `"form"`? I find it to be less generic.